services/web/server/src/simcore_service_webserver/workspaces/errors.py

```diff
@@ -0,0 +1,24 @@
+from ..errors import WebServerBaseError
+
+
```

Comment:
I find the errors in this module lacking. I much prefer the style of errors which are explicit and only are given the context (parameters required to render them).

Reply:
the style you are using also causes a lot of repetition.

Reply:
I think for the first version this is sufficient, additional iterations will be done.